variables.tf

```diff
@@ -25,6 +25,18 @@ variable "iam_service_id_tags" {
   default     = []
 }
 
+variable "iam_service_id_apikey_provision" {
+  description = "Enable to create an IAM service ID API key."
```

Comment:
should we have a question here?
similar as we have https://github.com/terraform-ibm-modules/terraform-ibm-observability-instances/blob/b9cf10a6686f50dde079f6f72a89cdcdabcd0aa0/variables.tf#L52